src/NoteField.cpp

```diff
@@ -867,7 +867,15 @@ void NoteField::DrawPrimitives()
 		ASSERT(GAMESTATE->m_pCurSong != nullptr);
 
 		const TimingData &timing = *pTiming;
-		const RageColor text_glow= RageColor(1,1,1,std::cos(RageTimer::GetTimeSinceStartFast()*2)/2+0.5f);
+
+		// Create an oscillating / pulsing glow effect.
+		// Converts a counter to radians and uses the cosine for a cyclic appearance.
+		static std::uint_fast16_t m_iGlowCounter;
```

Comment:
this doesn't need the `m_` prefix. It isn't a member variable.

Reply:
Oops, thanks for catching this!